lib/opensusebasetest.pm

```diff
@@ -329,8 +329,13 @@ sub wait_boot {
 
     # Reset the consoles after the reboot: there is no user logged in anywhere
     reset_consoles;
+
+    if (check_var('ARCH', 'ppc64le')) {
+        # give ppc another 120 seconds since it's slow sometimes
+        $ready_time += 120;
```

Comment:
Please see my comment on the progress ticket

Reply:
Do we already have a bug for this? Then I would add a softfail